### ISSUE TITLE
The timing of Train and Test sets

### ISSUE THREAD

**Issue by MiladAbdullah** [MiladAbdullah/phoenix#15]
According to our previous study, we know that the test set should not be `too far (in time)` from the training set. Usually, a window of 5-6 months keeps the accuracy of the model high. 

![image](https://github.com/MiladAbdullah/phoenix/assets/21346954/28a9344d-f0cb-419b-bc47-98a0de95d186)
